config/locales/mailers/en.yml

```diff
@@ -209,21 +227,24 @@ en:
         part10: "If you had other works on the imported archive under an e-mail address you can no longer access, please contact Open Doors with any information that can help verify your identity."
         part11: "For other inquiries, please contact AO3 Support at %{support_link}."
       html:
-        part1: "You're receiving this e-mail because an archive has recently been imported by %{open_doors_link} into the %{app_link} (%{app_short_name}), and we believe that the following fanworks belong to you. We'd like to give you a chance to claim (or delete/orphan) these works if you want to. And if you don't already have an account under a different e-mail, we'd like to invite you aboard!"
-        open_doors: "Open Doors"
-        part2: "You can read announcements about recent archive moves at %{news_link}, and find additional information on Open Doors' %{open_doors_faq_link} or %{open_doors_tutorial_link}. For any questions not answered in the FAQ, tutorials, or this e-mail, please %{support_link}."
+        part1: "You're receiving this e-mail because an archive has recently been imported by %{open_doors_name_link} into the %{app_link} (%{app_short_name}), and we believe that the following fanworks belong to you. We'd like to give you a chance to claim (or delete/orphan) these works if you want to. And if you don't already have an account under a different e-mail, we'd like to invite you aboard!"
+        open_doors_name: "Open Doors"
+        part2: "You can read announcements about recent archive moves at %{ao3_news_link}, and find additional information on Open Doors' %{faq_page_link} or %{tutorial_page_link}. For any questions not answered in the FAQ, tutorials, or this e-mail, please %{contact_support_link}."
         ao3_news: "AO3 News"
         faq_page: "FAQ page"
         tutorial_page: "tutorial page"
-        contact_support: "contact AO3 Support"
-        part3: "If this is a mistake and these are not your works, please don't delete them! Please just %{open_doors_link} and we will sort it out."
+        contact_support: "contact AO3 support"
```

Comment:
Can you leave the S capitalized so it matches the text version (and the other OD emails)? (#4074)

Reply:
Ah yes, missed that in the merge conflict, will do